test/integration/controllermanager/shoot/maintenance/maintenance_test.go

```diff
@@ -98,7 +105,72 @@ var _ = Describe("Shoot Maintenance controller tests", func() {
 						Versions: []gardencorev1beta1.MachineImageVersion{
 							{
 								ExpirableVersion: gardencorev1beta1.ExpirableVersion{
-									Version:        highestAMD64MachineImageVersion,
+									// ARM overall latest version: 3.0.0
+									Version:        overallLatestVersionARM,
+									Classification: &deprecatedClassification,
+								},
+								CRI: []gardencorev1beta1.CRI{
+									{
+										Name: gardencorev1beta1.CRINameDocker,
+									},
+									{
+										Name: gardencorev1beta1.CRINameContainerD,
+									},
+								},
+								Architectures: []string{"arm64"},
+							},
+							{
+								ExpirableVersion: gardencorev1beta1.ExpirableVersion{
+									// AMD overall latest version: 3.0.1
```

Comment:
Either make `AMD64` or `AMD` all the place, as of now some places it is `AMD` and someplace it is `AMD64`.
```suggestion
									// AMD64 overall latest version: 3.0.1
```